Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Android/Setup.cs

```diff
@@ -19,19 +27,29 @@ protected override IMvxIoCProvider CreateIocProvider()
         {
             var containerBuilder = new ContainerBuilder();
             containerBuilder.RegisterModule<ApplicationModule>();
+            containerBuilder.Register(c => Environment.GetFolderPath(Environment.SpecialFolder.Personal)).Named<string>(App.BootstrapParamaters.LocalAppDataFolder);
```

Comment:
So we have:

`containerBuilder.Register(c => {value}).Named<string>({name});`

I'm registering the value `Environment.GetFolderPath(Environment.SpecialFolder.Personal)`, and tagging it with the name `App.BootstrapParamaters.LocalAppDataFolder`

So, I'm saying `LocalAppDataFolder` path for android is `Environment.GetFolderPath(Environment.SpecialFolder.Personal)` value.

Then I'm using that value when creating a `LocalStorageContext` referencing it with

`c.ResolveNamed<string>(name: App.BootstrapParamaters.LocalAppDataFolder)`

>Is this parameter the name given to the registration? If so, maybe we should go with a constant value?

The name for the parameter is a constant value. Are you seeing something weird on this code?